src/widgets/latest-blocks/LatestBlocks.vue

```diff
@@ -22,9 +25,8 @@
 
             <div class="latest-blocks__time">
               <TimeIcon class="latest-blocks__time-icon" />
-              {{ $t('time.min', [elapsed.allMinutes(block.created_at)]) }}
-              {{ $t('time.sec', [elapsed.seconds(block.created_at)]) }}
-              {{ $t('time.ago') }}
+              <TimeAgo :value="block.created_at" />
+              <Tooltip :message="defaultFormat(block.created_at)" />
```

Comment:
Looks strange - tooltip should be attached to the "time ago" value, shouldn't it?

```html
<tooltip>
  <time-ago :value />
  <template #message> ... </template>
</tooltip>
```

Btw, seems like reasonable to be wrapped in a component which you had removed, i.e. `TimeStamp` (both tooltip and time ago inside)